addons/html_editor/static/tests/link/popover.test.js

```diff
@@ -499,6 +499,18 @@ describe("Link formatting in the popover", () => {
 });
 
 describe("shortcut", () => {
+    test("create link shortcut should be at the first", async () => {
+        const { editor } = await setupEditor(`<p>[]</p>`);
+        const remove_test_command = editor.services.command.add("A test command", () => {}, {
+            hotkey: "alt+k",
+            category: "app",
+        });
+
+        press(["ctrl", "k"]);
+        await animationFrame();
+        expect(queryText(".o_command_name:first")).toBe("Create link");
+        remove_test_command();
```

Comment:
You don t need to remove it. By default each service is destroy and restart for each test